tests/Functional/Action/Lti/GetUserAssignmentLtiLinkActionTest.php

```diff
@@ -62,14 +62,14 @@ protected function setUp(): void
 
     public function testItReturns401IfNotAuthenticated(): void
     {
-        $this->kernelBrowser->request('GET', '/api/v1/assignments/1/lti-link');
+        $this->kernelBrowser->request('GET', '/api/v1/assignments/00000001-0000-6000-0000-000000000000/lti-link');
```

Comment:
Maybe we could have a default property or constant for this UUID or for the entire URL, something like 
`$this->defaultLtiLink = '/api/v1/assignments/00000001-0000-6000-0000-000000000000/lti-link'`

Reply:
I don't know, there are different urls we are using here, not just only one, I'm not sure. Can we leave it as it is for now?

Reply:
Fine by me